src/operator-docs/combination/combineLatest.ts

```diff
@@ -0,0 +1,65 @@
+import { OperatorDoc } from '../operator.model';
+
+export const combineLatest: OperatorDoc = {
```

Comment:
Presumably this whole document is a stub and we're going to be generating this data from the TS files directly, right? 

We should be able to link the exact line (and SHA) in Github this doc was generated from so people can "read the source"

Reply:
Right, I wasn't sure about the long term solution. This is just a temporary hack so I could start getting data on the page / establish structure.